.github/workflows/build-e2e.yml

```diff
@@ -0,0 +1,37 @@
+name: Build
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  minikube-build:
+    name: Run smoke tests within minikube
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the code
+        uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ^1.18
+
+      - name: Install jsonnet
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y jsonnet
+
+      - name: Install jsonnet bundler
+        run: go install -a github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb@latest
+
+      - name: Install jsontoyaml
+        run: go get github.com/brancz/gojsontoyaml
+
+      - name: Start minikube
+        uses: medyagh/setup-minikube@latest
+
+      - name: Smoke tests
+        run: make test-e2e driver=docker
```

Comment:
this can't work, we need a real virtual machine for parca-agent to run on

Reply:
ditto